README.md

```diff
@@ -63,7 +65,7 @@ $connector = new oliverlorenz\reactphpmqtt\Connector($loop, $resolver, $version)
 
 $p = $connector->create($config['server'], $config['port'], $config['options']);
 $p->then(function(\React\Stream\Stream $stream) use ($connector) {
-    $stream->on('PUBLISH', function(\oliverlorenz\reactphpmqtt\packet\Publish $message) {
+    $stream->on(Publish::EVENT, function(Publish $message) {
```

Comment:
Can you put these variable/constant also to ControlPacket? :slightly_smiling_face: